templates/404.html

```diff
@@ -5,10 +5,20 @@
 {% block content %}
 <div class="p-strip">
   <div class="row">
-    <div class="col-6">
-      <h1>404: Page not found</h1>
-      <p class="p-notification__response">
-        {{ error }}
+    <div class="col-10">
+      <h2>404: Aw, snap! We couldn't find that page</h2>
+      <h4>Looking for a snap? Try searching:</h4>
+      <form action="/search" class="p-form p-form--inline p-form--search">
+     <div class="p-form__group">
+        <label for="search-input" class="u-off-screen">Search</label>
+        <div class="p-form__control u-clearfix">
+          <input class="u-no-margin--bottom" id="search-input" type="search" name="q" value="">
+        </div>
+      </div>
+      <button type="submit" alt="search" class="p-button--positive">Search</button>
+    </form>
+      <p>
```

Comment:
Yo dawg, I heard you liked `<p>`'s so I wrapped your `<p>` in a `<p>` so you could paragraph while you paragraph.

You should remove the outer `<p></p>`